components/notifications-service/server/lib/dispatcher.ex

```diff
@@ -6,6 +6,10 @@ defmodule Notifications.Dispatcher do
   require Logger
   use GenServer
 
+  def init(init_arg) do
+    {:ok, init_arg}
+  end
```

Comment:
What is the reason this was added?

Reply:
To fix a warning from notifications_lint 

`warning: function init/1 required by behaviour GenServer is not implemented (in module Notifications.Dispatcher).

We will inject a default implementation for now:

    def init(init_arg) do
      {:ok, init_arg}
    end

You can copy the implementation above or define your own that converts the arguments given to GenServer.start_link/3 to the server state.

  lib/dispatcher.ex:1
`